archaius-aws/src/main/java/com/netflix/config/sources/AbstractDynamoDbConfigurationSource.java

```diff
@@ -41,12 +45,16 @@
     static final String keyAttributePropertyName = "com.netflix.config.dynamo.keyAttributeName";
     static final String valueAttributePropertyName = "com.netflix.config.dynamo.valueAttributeName";
     static final String endpointPropertyName = "com.netflix.config.dynamo.endpoint";
+    static final String pollingMaxBackOff = "com.netflix.config.dynamo.maxPollingBackOff";
```

Comment:
I would also clarify that this is the property name not the property property object.  Notice how the other names are clearly labeled as PropertyName.